### PR TITLE
new package: zathura at 0.4.4

### PR DESCRIPTION
**Pull request by tristelune1**
## Description of package

Zathura pdf viewer. It's a lightweight pdf viewer with vim keybindings. 

## Checklist

- [x] Latest upstream version.
- [x] I agree to maintain this package (**required**).
- [x] I agree to notify KISS if I can no longer maintain this package (**required**).

### CODE REVIEW

**Comment by tristelune1** [comment]
Sorry, there is a problem. I'll have a look if I can modify my pull request. Two files are not installed.

**Comment by tristelune1** [comment]
Changes done. For me you can commit. Sorry for the mess, next time I'll try to make proper pull requests, I understand now better how it works!

**Comment by dylanaraps** [comment]
I would much rather we exclude the two man-pages and save the 2000 lines of additions. Why are they included?

**Comment by tristelune1** [comment]
I can search the information on internet, but sometimes I forgot a parameter and I don't have an internet connection. Normally the man page are not deleted as far as I know.

The problem with `zathura` is the man pages are built with `python-sphinx`. If I want to build them, I have to package `python-sphinx` and probably 5-10 pythons modules. A package maintainer for Crux has packaged everything and he includes directly those man pages such that the users don't have to install `python-sphinx` and all dependencies.

File: community/zathura/build [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#!/bin/sh -e
+
+patch -p1 < remove_gettext.patch
+
+meson build \
+	--prefix=/usr \
```
Comment (dylanaraps) [inline community/zathura/build]: 4 spaces for indentation.

**Comment by tristelune1** [comment]
Done!

**Comment by tristelune1** [comment]
Backend parameter removed from build.

File: community/zathura/build [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+meson build \
+    --prefix=/usr \
+    --sysconfdir=/etc \
+    --buildtype=release \
+    -Dsynctex=disabled
+ 
+DESTDIR="$1" ninja -C build install
+
+install -D -m 0644 zathura.1 "$1"/usr/share/man/man1/zathura.1
```
Comment (dylanaraps) [inline community/zathura/build]: Align it like this:

```sh
install -D -m 0644 zathura.1   "$1"/usr/share/man/man1/zathura.1
install -D -m 0644 zathurarc.5 "$1"/usr/share/man/man1/zathurarc.5
```

Comment (tristelune1) [reply community/zathura/build]: Done!

**Comment by dylanaraps** [comment]
The patch seems to be missing from this PR.

**Comment by tristelune1** [comment]
That's right. Locally I have this patch but I forgot it in this PR. Change done, thanks!

**Pull request merged**